htdocs/app/view/login/Readme.md

```diff
@@ -0,0 +1 @@
+This folder contains the Login view and controller.
```

Comment:
Consistency is irrelevant here; Readme files are used as placeholders so that Git won't delete empty directories. Once a directory is not empty and there are some meaningful files, Readme is no longer necessary.